synapse/lib/grammar.py

```diff
@@ -267,15 +267,6 @@ def switchcase(self, kids):
 
         return s_ast.SwitchCase(newkids)
 
-    def casevalu(self, kids):
```

Comment:
looks like this was unused?

Reply:
No, I just restructured the grammar so now lark does the work.

Reply:
awesome :) :muscle: